jenkins/deduper.py

```diff
@@ -143,12 +143,12 @@ def stop_duplicates(self, build_data):
                 # Now for each author, stop all but the most
                 # recent build.
                 for build_list in pr_builds.itervalues():
-                    output = self.stop_all_but_most_recent(pr, build_list)
+                    output = self.stop_all_but_most_recent(pull_request, build_list)
```

Comment:
oh snaps, now this line's too long.

$ pep8 .
./jenkins/deduper.py:146:80: E501 line too long (84 > 79 characters)